src/daycare.c

```diff
@@ -549,8 +545,11 @@ void BuildEggMoveset(struct Pokemon *egg, struct BoxPokemon *father, struct BoxP
             }
         }
         else
+        {
```

Comment:
Is this another scope declaration meme?

Reply:
No, I just added that because I felt like it.

Reply:
There weren't any strange fixes in this PR--just standard non-matching stuff.